server/util/db/db.go

```diff
@@ -247,7 +248,9 @@ func openDB(configurator *config.Configurator, dialect string, connString string
 	}
 
 	instrumentGORM(gdb)
-
+	if err := gdb.Use(otelgorm.NewPlugin()); err != nil {
```

Comment:
it looks like by default it will store variable values in the span, which can contain sensitive data. 
there's an WithoutQueryVariables option to disable this.